### PR TITLE
TF Lite is_hybrid_op check move to util

### PR DESCRIPTION
**Pull request by Dayananda-V**
is_hybrid_op generic check move to kernel_util file (ToDo activity).

### CODE REVIEW

File: tensorflow/lite/kernels/bidirectional_sequence_lstm.cc [hunk 1]
```diff
@@ -508,12 +508,8 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   TF_LITE_ENSURE_OK(context,
                     context->ResizeTensor(context, fw_output, fw_output_size));
 
-  // The weights are of consistent type, so it suffices to check one.
-  const bool is_hybrid_op = (fw_input_to_output_weights->type == kTfLiteUInt8 ||
-                             fw_input_to_output_weights->type == kTfLiteInt8);
```
Comment (haozha111) [inline tensorflow/lite/kernels/bidirectional_sequence_lstm.cc]: It seems this check is different from the others, without checking that input type at the same time. Could you confirm whether this is intended?

Comment (Dayananda-V) [reply tensorflow/lite/kernels/bidirectional_sequence_lstm.cc]: @haozha111  
L399 this check verification completed, so to keep consistency of method call using utility method but will not affect the existing logic.

`  const TfLiteTensor* input = GetInput(context, node, kInputTensor);
  TF_LITE_ENSURE_EQ(context, input->type, kTfLiteFloat32);
`

File: tensorflow/lite/kernels/bidirectional_sequence_lstm.cc [hunk 2]
```diff
@@ -601,7 +597,7 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   }
   TF_LITE_ENSURE_OK(context, context->ResizeTensor(context, bw_scratch_buffer,
                                                    bw_scratch_buffer_size));
-  if (is_hybrid_op) {
+  if (IsHybridOp(input, fw_input_to_output_weights)) {
```
Comment (haozha111) [inline tensorflow/lite/kernels/bidirectional_sequence_lstm.cc]: I think we just need call IsHybridOp once and store the result in is_hybrid_op, like the previous code do.

File: tensorflow/lite/kernels/lstm.cc [hunk 3]
```diff
@@ -380,14 +380,8 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
...
 
-  // The weights are of consistent type, so it suffices to check one.
-  // TODO(mirkov): create a utility/macro for this check, so all Ops can use it.
-  const bool is_hybrid_op = ((input_to_output_weights->type == kTfLiteUInt8 ||
-                              input_to_output_weights->type == kTfLiteInt8) &&
-                             input->type == kTfLiteFloat32);
-
   TfLiteIntArrayFree(node->temporaries);
-  if (is_hybrid_op) {
+  if (IsHybridOp(input, input_to_output_weights)) {
```
Comment (haozha111) [inline tensorflow/lite/kernels/lstm.cc]: same here, we just need to check it once.

File: tensorflow/lite/kernels/svdf.cc [hunk 4]
```diff
@@ -175,14 +175,9 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
...
 
-  // The weights are of consistent type, so it suffices to check one.
-  const bool is_hybrid_op = (input->type == kTfLiteFloat32 &&
-                             (weights_feature->type == kTfLiteUInt8 ||
-                              weights_feature->type == kTfLiteInt8));
-
   // Resize scratch.
   TfLiteIntArrayFree(node->temporaries);
-  if (is_hybrid_op) {
+  if (IsHybridOp(input, weights_feature)) {
```
Comment (haozha111) [inline tensorflow/lite/kernels/svdf.cc]: same here.

**Comment by Dayananda-V** [comment]
@haozha111 

Thanks for review comments, handle the comments , please have check one more time, TIA!

**Review by haozha111 — APPROVED** [review]

**Pull request merged**